package/gluon-ebtables-source-filter/files/lib/gluon/ebtables/110-source-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+site = require('gluon.site_config')
+
+rule('SOURCE -p IPv6 --ip6-src fe80::/10 -j RETURN')
```

Comment:
I guess this can be changed to fe80::/64, no other link-local addresses should be used on Ethernet-like links like the mesh.